src/v2/Apps/Settings/Routes/EditProfile/UserInformation.tsx

```diff
@@ -138,35 +149,27 @@ export const UserInformation: React.FC<UserInformationProps> = ({
             <input name="internalID" value={me.internalID} hidden readOnly />
             {touched.email && values.email !== me.email && (
               <>
-                <Text>Password</Text>
                 <PasswordInput
```

Comment:
Palette has a password input we'll want to use https://palette-storybook.artsy.net/?path=/story/components-passwordinput--default

Reply:
Utilized the PasswordInput, let me know if that was the one you meant.

Reply:
It should be coming from Palette, not Force

Reply:
Updated.